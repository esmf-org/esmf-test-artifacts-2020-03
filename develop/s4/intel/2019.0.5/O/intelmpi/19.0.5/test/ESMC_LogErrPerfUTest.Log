build time -- 2022-02-27 06:27:23
20220227 063729.482 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.482 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220227 063729.482 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220227 063729.482 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220227 063729.482 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220227 063729.482 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.482 INFO             PET0 Running with ESMF Version   : v8.3.0b08-5-g64eb133
20220227 063729.482 INFO             PET0 ESMF library build date/time: "Feb 27 2022" "06:25:11"
20220227 063729.482 INFO             PET0 ESMF library build location : /scratch/users/mpotts/intel_2019.0.5_intelmpi_O_develop
20220227 063729.482 INFO             PET0 ESMF_COMM                   : intelmpi
20220227 063729.483 INFO             PET0 ESMF_MOAB                   : enabled
20220227 063729.483 INFO             PET0 ESMF_LAPACK                 : enabled
20220227 063729.483 INFO             PET0 ESMF_NETCDF                 : enabled
20220227 063729.483 INFO             PET0 ESMF_PNETCDF                : disabled
20220227 063729.483 INFO             PET0 ESMF_PIO                    : enabled
20220227 063729.483 INFO             PET0 ESMF_YAMLCPP                : enabled
20220227 063729.483 INFO             PET0 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
20220227 063729.483 INFO             PET0 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
20220227 063729.484 INFO             PET0 perfFoundError: 1000	 iterations took 8.367e-06	 seconds. => 8.367e-09	 per iteration.
20220227 063729.484 INFO             PET0 PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
20220227 063729.484 INFO             PET0 perfFoundError: 10000	 iterations took 3.94993e-05	 seconds. => 3.94993e-09	 per iteration.
20220227 063729.484 INFO             PET0 PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
20220227 063729.484 INFO             PET0 perfFoundError: 100000	 iterations took 0.000399776	 seconds. => 3.99776e-09	 per iteration.
20220227 063729.484 INFO             PET0 PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
20220227 063729.488 INFO             PET0 perfFoundError: 1000000	 iterations took 0.00367432	 seconds. => 3.67432e-09	 per iteration.
20220227 063729.488 INFO             PET0 PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
20220227 063729.488 INFO             PET0 PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
20220227 063729.488 INFO             PET0 perfMsgFoundError: 1000	 iterations took 4.87641e-06	 seconds. => 4.87641e-09	 per iteration.
20220227 063729.488 INFO             PET0 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
20220227 063729.488 INFO             PET0 perfMsgFoundError: 10000	 iterations took 4.21107e-05	 seconds. => 4.21107e-09	 per iteration.
20220227 063729.488 INFO             PET0 PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
20220227 063729.489 INFO             PET0 perfMsgFoundError: 100000	 iterations took 0.000424378	 seconds. => 4.24378e-09	 per iteration.
20220227 063729.489 INFO             PET0 PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
20220227 063729.493 INFO             PET0 perfMsgFoundError: 1000000	 iterations took 0.00472709	 seconds. => 4.72709e-09	 per iteration.
20220227 063729.493 INFO             PET0 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
20220227 063729.493 INFO             PET0 PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
20220227 063729.493 INFO             PET0 Ending Test, file ESMC_LogErrPerfUTest.C, line 176
20220227 063729.493 INFO             PET0 Finalizing ESMF
20220227 063729.483 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.483 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220227 063729.483 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220227 063729.483 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220227 063729.483 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220227 063729.483 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.483 INFO             PET1 Running with ESMF Version   : v8.3.0b08-5-g64eb133
20220227 063729.483 INFO             PET1 ESMF library build date/time: "Feb 27 2022" "06:25:11"
20220227 063729.483 INFO             PET1 ESMF library build location : /scratch/users/mpotts/intel_2019.0.5_intelmpi_O_develop
20220227 063729.483 INFO             PET1 ESMF_COMM                   : intelmpi
20220227 063729.484 INFO             PET1 ESMF_MOAB                   : enabled
20220227 063729.484 INFO             PET1 ESMF_LAPACK                 : enabled
20220227 063729.484 INFO             PET1 ESMF_NETCDF                 : enabled
20220227 063729.484 INFO             PET1 ESMF_PNETCDF                : disabled
20220227 063729.484 INFO             PET1 ESMF_PIO                    : enabled
20220227 063729.484 INFO             PET1 ESMF_YAMLCPP                : enabled
20220227 063729.484 INFO             PET1 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
20220227 063729.484 INFO             PET1 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
20220227 063729.484 INFO             PET1 perfFoundError: 1000	 iterations took 6.61612e-06	 seconds. => 6.61612e-09	 per iteration.
20220227 063729.484 INFO             PET1 PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
20220227 063729.484 INFO             PET1 perfFoundError: 10000	 iterations took 3.9462e-05	 seconds. => 3.9462e-09	 per iteration.
20220227 063729.484 INFO             PET1 PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
20220227 063729.485 INFO             PET1 perfFoundError: 100000	 iterations took 0.000393931	 seconds. => 3.93931e-09	 per iteration.
20220227 063729.485 INFO             PET1 PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
20220227 063729.488 INFO             PET1 perfFoundError: 1000000	 iterations took 0.00363339	 seconds. => 3.63339e-09	 per iteration.
20220227 063729.488 INFO             PET1 PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
20220227 063729.488 INFO             PET1 PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
20220227 063729.488 INFO             PET1 perfMsgFoundError: 1000	 iterations took 4.85033e-06	 seconds. => 4.85033e-09	 per iteration.
20220227 063729.488 INFO             PET1 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
20220227 063729.489 INFO             PET1 perfMsgFoundError: 10000	 iterations took 4.21144e-05	 seconds. => 4.21144e-09	 per iteration.
20220227 063729.489 INFO             PET1 PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
20220227 063729.489 INFO             PET1 perfMsgFoundError: 100000	 iterations took 0.000420444	 seconds. => 4.20444e-09	 per iteration.
20220227 063729.489 INFO             PET1 PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
20220227 063729.494 INFO             PET1 perfMsgFoundError: 1000000	 iterations took 0.0045845	 seconds. => 4.5845e-09	 per iteration.
20220227 063729.494 INFO             PET1 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
20220227 063729.494 INFO             PET1 PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
20220227 063729.494 INFO             PET1 Ending Test, file ESMC_LogErrPerfUTest.C, line 176
20220227 063729.494 INFO             PET1 Finalizing ESMF
20220227 063729.483 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.483 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220227 063729.483 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220227 063729.483 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220227 063729.483 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220227 063729.483 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.483 INFO             PET2 Running with ESMF Version   : v8.3.0b08-5-g64eb133
20220227 063729.483 INFO             PET2 ESMF library build date/time: "Feb 27 2022" "06:25:11"
20220227 063729.483 INFO             PET2 ESMF library build location : /scratch/users/mpotts/intel_2019.0.5_intelmpi_O_develop
20220227 063729.483 INFO             PET2 ESMF_COMM                   : intelmpi
20220227 063729.485 INFO             PET2 ESMF_MOAB                   : enabled
20220227 063729.485 INFO             PET2 ESMF_LAPACK                 : enabled
20220227 063729.485 INFO             PET2 ESMF_NETCDF                 : enabled
20220227 063729.485 INFO             PET2 ESMF_PNETCDF                : disabled
20220227 063729.485 INFO             PET2 ESMF_PIO                    : enabled
20220227 063729.485 INFO             PET2 ESMF_YAMLCPP                : enabled
20220227 063729.485 INFO             PET2 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
20220227 063729.485 INFO             PET2 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
20220227 063729.485 INFO             PET2 perfFoundError: 1000	 iterations took 1.81161e-05	 seconds. => 1.81161e-08	 per iteration.
20220227 063729.485 INFO             PET2 PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
20220227 063729.485 INFO             PET2 perfFoundError: 10000	 iterations took 0.000110555	 seconds. => 1.10555e-08	 per iteration.
20220227 063729.485 INFO             PET2 PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
20220227 063729.486 INFO             PET2 perfFoundError: 100000	 iterations took 0.00112449	 seconds. => 1.12449e-08	 per iteration.
20220227 063729.486 INFO             PET2 PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
20220227 063729.491 INFO             PET2 perfFoundError: 1000000	 iterations took 0.00427926	 seconds. => 4.27926e-09	 per iteration.
20220227 063729.491 INFO             PET2 PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
20220227 063729.491 INFO             PET2 PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
20220227 063729.491 INFO             PET2 perfMsgFoundError: 1000	 iterations took 5.87106e-06	 seconds. => 5.87106e-09	 per iteration.
20220227 063729.491 INFO             PET2 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
20220227 063729.491 INFO             PET2 perfMsgFoundError: 10000	 iterations took 4.21107e-05	 seconds. => 4.21107e-09	 per iteration.
20220227 063729.491 INFO             PET2 PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
20220227 063729.491 INFO             PET2 perfMsgFoundError: 100000	 iterations took 0.000420399	 seconds. => 4.20399e-09	 per iteration.
20220227 063729.491 INFO             PET2 PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
20220227 063729.496 INFO             PET2 perfMsgFoundError: 1000000	 iterations took 0.00423397	 seconds. => 4.23397e-09	 per iteration.
20220227 063729.496 INFO             PET2 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
20220227 063729.496 INFO             PET2 PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
20220227 063729.496 INFO             PET2 Ending Test, file ESMC_LogErrPerfUTest.C, line 176
20220227 063729.496 INFO             PET2 Finalizing ESMF
20220227 063729.482 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.482 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220227 063729.482 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220227 063729.482 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220227 063729.482 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220227 063729.482 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220227 063729.482 INFO             PET3 Running with ESMF Version   : v8.3.0b08-5-g64eb133
20220227 063729.482 INFO             PET3 ESMF library build date/time: "Feb 27 2022" "06:25:11"
20220227 063729.482 INFO             PET3 ESMF library build location : /scratch/users/mpotts/intel_2019.0.5_intelmpi_O_develop
20220227 063729.482 INFO             PET3 ESMF_COMM                   : intelmpi
20220227 063729.483 INFO             PET3 ESMF_MOAB                   : enabled
20220227 063729.483 INFO             PET3 ESMF_LAPACK                 : enabled
20220227 063729.483 INFO             PET3 ESMF_NETCDF                 : enabled
20220227 063729.483 INFO             PET3 ESMF_PNETCDF                : disabled
20220227 063729.483 INFO             PET3 ESMF_PIO                    : enabled
20220227 063729.483 INFO             PET3 ESMF_YAMLCPP                : enabled
20220227 063729.483 INFO             PET3 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
20220227 063729.483 INFO             PET3 Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
20220227 063729.484 INFO             PET3 perfFoundError: 1000	 iterations took 7.00727e-06	 seconds. => 7.00727e-09	 per iteration.
20220227 063729.484 INFO             PET3 PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
20220227 063729.484 INFO             PET3 perfFoundError: 10000	 iterations took 3.95104e-05	 seconds. => 3.95104e-09	 per iteration.
20220227 063729.484 INFO             PET3 PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
20220227 063729.484 INFO             PET3 perfFoundError: 100000	 iterations took 0.000398703	 seconds. => 3.98703e-09	 per iteration.
20220227 063729.484 INFO             PET3 PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
20220227 063729.488 INFO             PET3 perfFoundError: 1000000	 iterations took 0.00366931	 seconds. => 3.66931e-09	 per iteration.
20220227 063729.488 INFO             PET3 PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
20220227 063729.488 INFO             PET3 PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
20220227 063729.488 INFO             PET3 perfMsgFoundError: 1000	 iterations took 5.84126e-06	 seconds. => 5.84126e-09	 per iteration.
20220227 063729.488 INFO             PET3 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
20220227 063729.488 INFO             PET3 perfMsgFoundError: 10000	 iterations took 4.21032e-05	 seconds. => 4.21032e-09	 per iteration.
20220227 063729.488 INFO             PET3 PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
20220227 063729.489 INFO             PET3 perfMsgFoundError: 100000	 iterations took 0.000479154	 seconds. => 4.79154e-09	 per iteration.
20220227 063729.489 INFO             PET3 PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
20220227 063729.493 INFO             PET3 perfMsgFoundError: 1000000	 iterations took 0.00436982	 seconds. => 4.36982e-09	 per iteration.
20220227 063729.493 INFO             PET3 PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
20220227 063729.493 INFO             PET3 PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
20220227 063729.493 INFO             PET3 Ending Test, file ESMC_LogErrPerfUTest.C, line 176
20220227 063729.493 INFO             PET3 Finalizing ESMF
