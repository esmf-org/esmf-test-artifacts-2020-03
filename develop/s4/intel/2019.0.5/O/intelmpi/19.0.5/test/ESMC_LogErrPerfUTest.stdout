build time -- 2022-02-27 06:27:23
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88
Beginning Test, file ESMC_LogErrPerfUTest.C, line 88NUMBER_OF_PROCESSORS 4
PASS Performance of ESMCI::LogErr::FoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 96
PASS Performance of ESMCI::LogErr::FoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 104
PASS Performance of ESMCI::LogErr::FoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 112
PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
PASS Performance of ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 120
PASS Threshold check for ESMCI::LogErr::FoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 130
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000x Test, ESMC_LogErrPerfUTest.C, line 138
PASS Performance of ESMCI::LogErr::MsgFoundError() 10000x Test, ESMC_LogErrPerfUTest.C, line 146
PASS Performance of ESMCI::LogErr::MsgFoundError() 100000x Test, ESMC_LogErrPerfUTest.C, line 154
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
Ending Test, file ESMC_LogErrPerfUTest.C, line 176
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
Ending Test, file ESMC_LogErrPerfUTest.C, line 176
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
Ending Test, file ESMC_LogErrPerfUTest.C, line 176
PASS Performance of ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 162
PASS Threshold check for ESMCI::LogErr::MsgFoundError() 1000000x Test, ESMC_LogErrPerfUTest.C, line 172
Ending Test, file ESMC_LogErrPerfUTest.C, line 176
Ending Test, file ESMC_LogErrPerfUTest.C, line 176 PET 1 Test Elapsed Time 13.315 msec.
Ending Test, file ESMC_LogErrPerfUTest.C, line 176 PET 0 Test Elapsed Time 14.408 msec.
Ending Test, file ESMC_LogErrPerfUTest.C, line 176 PET 2 Test Elapsed Time 12.788 msec.
Ending Test, file ESMC_LogErrPerfUTest.C, line 176 PET 3 Test Elapsed Time 15.498 msec.
TOTCPU=05:23:44 ELAP=00:10:07 REQMEM=5875Mc REQCPUS=24 ALLOCCPUS=32 TIMELIMIT=02:30:00 PART=s4 ACCT=star
MAXRSS=4020K MAXVMSIZE=196896K
________________________________________________________________
Job Resource Usage Summary for 12168021

  CPU Time Used : 05:23:44
  Memory Used : 4020K
  Virtual Memory Used : 196896K
  Walltime Used : 00:10:07

  Memory Requested : 5875Mc (n=per node; c=per core)
  CPUs Requested / Allocated : 24 / 32
  Walltime Requested : 02:30:00

  Execution Queue : s4
  Head Node : 
  Charged to : star

  Job Stopped : Sun Feb 27 06:37:30 UTC 2022
_____________________________________________________________________

